packages/core/src/middleware/koa-root-proxy.test.ts

```diff
@@ -0,0 +1,34 @@
+import { MountedApps } from '@/env-set';
+import { createContextWithRouteParameters } from '@/utils/test-utils';
+
+import koaRootProxy from './koa-root-proxy';
+
+describe('koaRootProxy', () => {
+  const next = jest.fn();
+
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should redirect to console', async () => {
```

Comment:
nit: the description could be more concrete 'should redirect to console when the path is empty'